<!DOCTYPE HTML>
<!--
	Theory by TEMPLATED
	templated.co @templatedco
	Released for free under the Creative Commons Attribution 3.0 license (templated.co/license)
-->
<html>

<head>
  <title>Research</title>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <link rel="stylesheet" href="assets/css/main.css" />
</head>

<body class="subpage">

  <!-- Header -->
  <section id="header">
    <header id="header">
      <div class="inner">
        <a href="index.html" class="logo">TISL</a>
        <nav id="nav">
          <a href="about.html">About Us</a>
          <a href="research.html">Research</a>
          <a href="education.html">Education</a>
          <a href="tools.html">Tools</a>
          <a href="join.html">Join Us</a>
          <a href="mailto:info@tislab.org">Contact</a>
        </nav>
        <a href="#navPanel" class="navPanelToggle"><span class="fa fa-bars"></span></a>
      </div>
    </header>
  </section>

  <!-- Main -->
  <section id="main" class="wrapper">
    <div class="inner">
      <header class="align-center">
        <h2>Current Research</h2>
      </header>

      <div class="row">
        <div class="4u">
          <img src="images/research/monarch-logo-black.png" width="100%" />
          <br />
          <br />
          <br />
          <p>
            <strong>Contact: </strong><a href="mailto:mcmurryj@oregonstate.edu">Julie McMurry.</a>
            <br><em>Funding:</em> Monarch is funded by NIH grant # 1R24OD011883-01 and NHGRI 1RM1HG010860-01
            <h2><a href="https://twitter.com/monarchinit"><i class="fa fa-twitter" aria-hidden="true"></i></a>
							<a href="https://github.com/monarch-initiative"><i class="fa fa-github" aria-hidden="true"></i></a>
							<a href="http://monarchinitiative.org"><i class="fa fa-globe" aria-hidden="true"></i></a></h2>
          </p>
        </div>
        <div class="8u">
          <h3><a href="https://monarchinitiative.org/">Monarch Initiative: Disease diagnosis and discovery</a></h3>

          <p>Over the last 20 years, the genomic revolution has provided tremendous amounts of knowledge in nearly all scientific fields, especially genomic medicine. But that data is complex, exists in different forms, and is distributed across different
            locations. It can also be incomplete or conflicting, or downright overwhelming to identify what parts of available knowledge are relevant to any given question.</p>

          <p>The challenges right now are not due to lack of data--there is a huge and growing amount of data. What we lack are tools to make effective use of that data for everyone. The Monarch Initiative was recently expanded through a $10M award to establish
            a Center of Excellence in Genomic Sciences (CEGS) and develop tools to address exactly this data problem. Through the CEGS project, scientists and physicians at four institutions in the U.S. and U.K are collaborating to develop those tools.</p>

          <p>Monarch is a collaboration among members at Oregon State University, Oregon Health & Science University, Lawrence Berkeley National Laboratory, The Jackson Laboratory For Genomic Medicine, RTI International, Genomics England/Queen Mary, Ada
            Health, and EMBL-EBI</p>


        </div>
      </div>

      <hr>
      <div class="row">
        <div class="8u">
          <h3>Phenomics-First Resource</h3>

          <p>The human genome has been sequenced, and yet so much is unknown about what it does; what we do know is scattered across multiple, heterogeneous data sources that are difficult to integrate. The Phenomics-First Resource (PFR) coordinates the
            efforts of a large community of researchers and clinicians to help interpret the relationship between the differences (called variants) in a patient’s genome and their physical characteristics (phenotypes). We are using the emerging field
            of phenomics to improve precision medicine and innovate in evolutionary biology. Patients with rare diseases will particularly benefit from this research; it takes an average of seven years to get a diagnosis for a rare disease, and even then
            the initial diagnosis is often wrong. Phenomics can be applied in a range of fields, but in this case the researchers are primarily focused on human disease and drawing insights from laboratory organisms.</p>

          <p>The PFR will gather and unify the data from numerous sources to advance genomic interpretation in clinical settings, and will empower both researchers and clinicians to overcome long-standing barriers to discovery and patient care. This project
            forms the basis of a Center of Excellence in Genomic Science, a program of the National Human Genome Research Institute (NHGRI), from the National Institutes of Health (NIH).</p>



        </div>
        <div class="4u">
          <img src="images/research/phenomics_first_funnel_population600.png" width="100%" />
          <br />
          <br />
          <p>
            <strong>Contact: </strong><a href="mailto:munoztmo@oregonstate.edu">Monica Munoz-Torres.</a>
            <br><em>Funding:</em> NHGRI
          </p>
        </div>
      </div>

      <hr>
      <div class="row">
        <div class="4u">
          <img src="images/research/globe-focus-tisc.png" width="100%" />
          <br />
          <br />
          <p>
            <strong>Contact: </strong><a href="mailto:thessena@oregonstate.edu">Anne Thessen.</a>
            <br><em>Funding:</em> This project is funded by the Office of Advanced Cyberinfrastructure (OAC), NSF Award #1940062.
            <h2><a href="https://twitter.com/GenoPhenoEnvo"><i class="fa fa-twitter" aria-hidden="true"></i></a>
							<a href="https://github.com/genophenoenvo"><i class="fa fa-github" aria-hidden="true"></i></a>
							<a href="https://genophenoenvo.github.io/"><i class="fa fa-globe" aria-hidden="true"></i></a></h2>
          </p>
        </div>
        <div class="8u">
          <h3>Converging Genomics, Phenomics, and Environments Using Interpretable Machine Learning Models</h3>

          <p>TISC is revolutionizing our understanding of how genes are transformed into organism traits in a changing environment. One of the central dogmas of biology is that genes and environments work together to determine the characteristics of an organism,
            or its phenotype. Yet, being able to predict with precision what an organism will look like or do based on its genes and environment has, for the most part, remained out of reach. Turns out, there’s a lot going on in a cell and in an organism
            that we just don’t understand which has a huge impact on how that organism looks and behaves. In addition to revolutionizing the way we think about how genes are expressed, we have to develop brand new ways of transforming and analyzing data
            so we can make best use of innovations in artificial intelligence and deep learning. TISC is leading this effort to innovate in computer science and biology to combat the negative effects of climate change on public health, conservation, and
            agriculture.
          </p>


        </div>
      </div>




      <hr>
      <div class="row">
        <div class="12u">
          <h3><a href="https://monarch-initiative.github.io/phenomics/">Forums for Integrative Phenomics</a></h3>

          <p>TISC is bursting through communication barriers to advance science. One of the biggest barriers to scientific advancement is lack of communication across different kinds of expertise, especially between researchers who study genetics, environmental
            science, and phenomics. This results in data that are totally incompatible with each other and blocks scientific progress. The goal of this project is to bring together a diverse range of research scientists and clinicians from these three
            areas to find ways to communicate and share data. Unified data enables new insights in relationships between diseases, genes, and environments in a wide range of species. TISC is creating new discoveries by planning and hosting workshops for
            creating new data standards that act as a “rosetta stone” between disciplines.</p>

        </div>
        <div class="4u">

          <p>
            <strong>Contact: </strong><a href="mailto:mcmurryj@oregonstate.edu">Julie McMurry.</a>
            <br><em>Funding:</em> This project is funded by NIH grant # 1 U13 CA221044-01.
          </p>
        </div>
      </div>


      <hr>
      <div class="row">
        <div class="8u">
          <h3>Center for Cancer Data Harmonization</h3>


          <p>
            TISC is leading development of a data-powered support infrastructure to cure and treat cancer. The data that researchers need to develop new life saving therapies spans disciplines and biological scales, especially where personalized approaches are needed.
            Currently, integrating data ranging from genomes and cellular components to patients and symptoms is a significant barrier to research.
          </p>
          <p>To address these issues TISC is taking a leading role in the Center for Cancer Data Harmonization (CCDH) whose mission is to bring data together across the various nodes of NCI’s Cancer Research Data Commons (CRDC) and make them accessible to
            researchers of any discipline in the fight against cancer.</p>

        </div>
        <div class="4u">
          <center><img src="images/research/ccdh_wheel.png" width="80%" /></center>
          <br />

          <p>
            <strong>Contact: </strong><a href="mailto:munoztmo@oregonstate.edu">Monica Munoz-Torres.</a>
            <br><em>Funding:</em> CCDH is funded by NCI / Leidos contract # HHSN261201500003I.
            <h2>
													<a href="https://github.com/cancerDHC"><i class="fa fa-github" aria-hidden="true"></i></a>
													<a href="https://datascience.cancer.gov/data-commons/center-cancer-data-harmonization-ccdh/"><i class="fa fa-globe" aria-hidden="true"></i></a></h2>
          </p>
        </div>
      </div>





      <hr>
      <div class="row">
        <div class="4u">
          <center><img src="images/research/gabriella-miller-kids-first-logo.png" width="80%" /></center>
          <br />
          <p>
            <strong>Contact: </strong><a href="mailto:vasilevs@ohsu.edu">Nicole Vasilevsky.</a>
            <br><em>Funding:</em> Kids First is funded by NIH grant # 5U2CHL138346 and The Kids First supplement is funded by NIH grant # 3200670520-03S1.
            <h2>
														<a href="https://twitter.com/kidsfirstDRC"><i class="fa fa-twitter" aria-hidden="true"></i></a>
														<a href="https://www.facebook.com/kidsfirstDRC/?ref=bookmarks"><i class="fa fa-facebook" aria-hidden="true"></i></a>
														<a href="https://www.youtube.com/channel/UCK9sPu0j4_ci4m3nNFa6gVw"><i class="fa fa-youtube" aria-hidden="true"></i></a>
														<a href="https://github.com/kids-first"><i class="fa fa-github" aria-hidden="true"></i></a>
														<a href="https://kidsfirstdrc.org/"><i class="fa fa-globe" aria-hidden="true"></i></a></h2>
          </p>
        </div>
        <div class="8u">
          <h3><a href="https://kidsfirstdrc.org/">Gabriella Miller Kids First Pediatric Data Resource Center</a></h3>
          <p>
            TISC is fighting pediatric cancer by supporting The Kids First Data Resource Portal, one of the largest collections of integrated genomic and clinical data for some childhood diseases, which previously were studied largely in isolation. Kids First brings
            together heterogeneous data from childhood cancers and structural birth defects to support research, study, and collaboration built on top of an unprecedented collection of genetic and phenotypic data from pediatric patients. The data in the
            Data Resource Portal is curated and structured using biomedical ontologies, including the Human Phenotype Ontology (HPO) and Mondo Disease Ontology to allow for query and retrieval of symptoms, findings and disease features across the cohorts.
            Additional work is being done in collaboration with the INCLUDE (INvestigation of Co-occurring conditions across the Lifespan to Understand Down syndromE) project to augment the Kids First Data Resource Center’s ability to integrate survey
            data for assessment of Down Syndrome-related clinical data and neurobehavior.
          </p>


        </div>
      </div>



      <hr>
      <div class="row">
        <div class="8u">
          <h3><a href="https://transltr.io/">Biomedical Data Translator Program</a></h3>
          <p>
            NCATS studies translation on a system-wide level as a scientific and operational problem. TISC convenes expert teams from diverse scientific disciplines — including, among others, efficacy, toxicity, data sharing, biomarkers and clinical application —
            to reduce, remove or bypass significant bottlenecks across the entire continuum of translation and to train the future translational science workforce.
          </p>

        </div>
        <div class="4u">
          <center><img src="images/research/translatorgraphic2020_600x300.jpg" width="100%" /></center>
          <br />

          <p>
            <strong>Contact: </strong><a href="mailto:mcmurryj@oregonstate.edu">Julie McMurry.</a>
            <br><em>Funding:</em> This project is funded by NCATS grant # 3 OT3 TR002019 01S2.
            <h2>
																										<a href="https://github.com/NCATSTranslator"><i class="fa fa-github" aria-hidden="true"></i></a>
																										<a href="https://ncats.nih.gov/translator/"><i class="fa fa-globe" aria-hidden="true"></i></a></h2>
          </p>
        </div>

      </div>


      <hr>
      <div class="row">
        <div class="4u">
          <center><img src="images/research/cd2h_logo.png" width="50%" /></center>
          <br />
          <p>
            <strong>Contact: </strong><a href="mailto:mcmurryj@oregonstate.edu">Julie McMurry.</a>
            <br><em>Funding:</em> Funding: This project is funded by NCATS grant # U24TR002306


            <h2>
														<a href="http://twitter.com/data2health"><i class="fa fa-twitter" aria-hidden="true"></i></a>
														<a href="https://github.com/data2health"><i class="fa fa-github" aria-hidden="true"></i></a>
														<a href="https://cd2h.org/"><i class="fa fa-globe" aria-hidden="true"></i></a></h2>
          </p>
        </div>
        <div class="8u">
          <h3>CD2H: Harmonizing the Informatics Community</h3>
          <p>
            TISC is leading the National Center for Data to Health (CD2H) using team science and translational research to improve patient care. CD2H accelerates advancements in informatics by using findable, accessible, interoperable, and reusable (FAIR) principles
            to promote collaboration across the Clinical and Translational Science Awards (CTSA) Program community. CD2H tools and resources make it simple and valuable for CTSA Program members to get engaged, connect with peers, and contribute. By promoting
            collaboration, CD2H fosters a robust translational science informatics ecosystem that collectively develops solutions to solve clinical problems faster, more efficiently, and more effectively. CTSA Program members are poised to lead this charge
            by harnessing collective expertise and strengths to solve key informatics challenges.
          </p>


        </div>
      </div>




      <hr>
      <div class="row">
        <div class="8u">
          <h3><a href="https://covid.cd2h.org/N3C">National COVID Cohort Collaborative (N3C)</a></h3>


          <p>
            The <a href="https://ncats.nih.gov/" style="text-decoration:none;">National Center for Advancing Translational Sciences (NCATS)</a> and the <a href="https://cd2h.org/" style="text-decoration:none;">National Center for Data to Health (CD2H)</a>            have created the <a href="https://covid.cd2h.org/" style="text-decoration:none;">National COVID Cohort Collaborative (N3C)</a> Data Enclave. The N3C aims to aggregate, harmonize, and make accessible vast amounts of data from healthcare providers
            nationwide to accelerate advances in COVID-19 research and clinical care. With the uncertainty of the COVID-19 global pandemic, the scientific community and <a href="https://ncats.nih.gov/ctsa" style="text-decoration:none;">Clinical and Translational Science Awards (CTSA) Program</a>            created the N3C partnership to overcome technical, regulatory, policy, and governance barriers to harmonizing and sharing individual-level clinical data.</p>

          <p>
            N3C welcomes researchers and citizen scientists to sign up and register for access to this nationwide, comprehensive health records dataset. Learn more at the <a href="https://covid.cd2h.org/" style="text-decoration:none;">CD2H N3C page</a>.</p>


        </div>
        <div class="4u">
          <center><img src="images/research/N3C_logo.png" width="90%" /></center>
          <br />

          <p>
            <strong>Contact: </strong><a href="mailto:melissa@tislab.org">Melissa Haendel.</a>
            <br><em>Funding:</em> National Institutes of Health's National Center for Advancing Translational Sciences, Grant Number U24TR002306.

            <h2>
																										<a href="http://twitter.com/N3C"><i class="fa fa-twitter" aria-hidden="true"></i></a>
																										<a href="https://github.com/National-COVID-Cohort-Collaborative"><i class="fa fa-github" aria-hidden="true"></i></a>
																										<a href="https://ncats.nih.gov/n3c/"><i class="fa fa-globe" aria-hidden="true"></i></a></h2>
          </p>
        </div>

      </div>







      <hr>
      <div class="row">
        <div class="4u">
          <center><img src="images/research/health-system-capacity.svg" width="90%" /></center>
          <br />

          <p>
            <strong>Contact: </strong><a href="mailto:mcmurryj@oregonstate.edu">Julie McMurry.</a>
            <br><em>Funding:</em> Funded by NIH U24TR002306.

            <h2>
										<a href="https://twitter.com/FlattenCurve"><i class="fa fa-twitter" aria-hidden="true"></i></a>
										<a href="https://github.com/flattenthecurve"><i class="fa fa-github" aria-hidden="true"></i></a>
										<a href="https://www.flattenthecurve.com/"><i class="fa fa-globe" aria-hidden="true"></i></a></h2>
          </p>
        </div>
        <div class="8u">
          <h3><a href="https://www.flattenthecurve.com/">Flatten the Curve</a></h3>
          <p>
            We can all slow the spread of COVID-19 and “Flatten the Curve” together. Flatten the Curve provides reliable information reviewed by scientists delivered in as many languages as possible to help.
          </p>


        </div>
      </div>

      <hr>
      <div class="row">
        <div class="8u">
          <h3>Integrated Health Sciences Facility Core</h3>
          <p>
            In collaboration with Oregon State’s <a href="https://ehsc.oregonstate.edu/" style="text-decoration:none;">Environmental Health Sciences Center</a> the <a href="https://lpi.oregonstate.edu/" style="text-decoration:none;">Linus Pauling Institute</a>,
            TISC is developing a new vision for the EHSC’s Integrated Health Sciences Facility Core, dedicated to connecting researchers and community stakeholders through technology and data. By building tools like an interactive, searchable Discovery
            Index to identify interest and expertise, we can quickly build cross-disciplinary teams to tackle new challenges, and connect existing teams to the communities their work impacts.</p>


        </div>
        <div class="4u">
          <center><img src="images/research/tehr-logo300.png" width="80%" /></center>

          <br />
          <p>
            <strong>Contact: </strong><a href="mailto:thessena@oregonstate.edu">Anne Thessen.</a>
            <br><em>Funding:</em> NIEHS 1P30ES030287-01A1.

            <h2>
							<a href="https://ehsc.oregonstate.edu/"><i class="fa fa-globe" aria-hidden="true"></i></a></h2>
          </p>
        </div>

      </div>



      <hr>
      <div class="row">
        <div class="12u">
          <h3>Food Innovation Group</h3>
          <p>
            TISC is providing critical data integration services to the new OSU Food Innovation Group. Food is central to all of our lives and in turn the choices that we make every day affect food and food production. The linear model of farm-store-consumer is not
            detailed or nuanced enough to understand how consumer decisions get made or what drives the food economy.
          </p>

          <p>
            <strong>Contact: </strong><a href="mailto:melissa@tislab.org">Melissa Haendel.</a>
            <br><em>Funding:</em>
          </p>
        </div>
      </div>


      <hr>
      <div class="row">
        <div class="12u">
          <h3>Environmental Health Sciences Vocabulary</h3>
          <p>
            TISC is laying the foundation for a new community-driven effort to build data standards for the environmental health sciences.
          </p>

          <p>
            <strong>Contact: </strong><a href="mailto:thessena@oregonstate.edu">Anne Thessen.</a>
            <br><em>Funding:</em> NIEHS
          </p>
        </div>
      </div>



			<br />
			<br />
			<br />
			<br />
			<br />

			<header class="align-center">
        <h2>Previous Research</h2>

      </header>									<h3><a href="https://undiagnosed.hms.harvard.edu/" class="logo">Undiagnosed Diseases Network (UDN)</a></h3>
									<p>We are part of the Metabolomics Core of the Undiagnosed Diseases Network (UDN). Our goals are to integrate metabolomics, lipidomics, glycomics, and genomics data with patient clinical phenotypes to provide mechanistic insight and aid diagnoses of rare and undiagnosed diseases. We are particularly involved in the integration of metabolites using existing pathway tools, reaction databases, and the integrated corpus of genotype-phenotype data within the Monarch platform for biological interpretation of disease etiology and biomarker signatures. We also started representing changes in glycomics signatures of patients with genetic diseases and undiagnosed diseases with the Molecular Glyco-Phenotype Ontology (MGPO) so as to enhance Human Phenotype Ontology and model data corpus to best leverage these phenotypic changes in the Exomiser tool. <b>Contact:</b> <a href="/cdn-cgi/l/email-protection#81ece2ecf4f3f3f8ebc1eef3e4e6eeeff2f5e0f5e4afe4e5f4">Julie McMurry</a>.

			<!need to add funding>

									<h3><a href="https://phenotypr.com/" class="logo">Phenotypr</a></h3>
									<p>Phenotypr is a free educational tool for people who believe they may have a disorder and want to learn more about their condition. This tool aims to provide additional information about what you are experiencing. We recommend you discuss this information with your healthcare provider to assist in your diagnosis. <b>Contact:</b> <a href="/cdn-cgi/l/email-protection#68050b051d1a1a110228071a0d0f07061b1c091c0d460d0c1d">Julie McMurry.</a><br><br>
									<i>Funding:</i> This project was funded by PCORI grant # 1R24OD011883-01.</p>

									<h3><a href="https://github.com/INCAtools/" class="logo">INCA Tools</a></h3>
									<p>This project aims to develop an intelligent concept assistant that will allow researchers to generate and share sets of metadata elements relevant to their project, and will use machine learning techniques to automatically apply this to data. <b>Contact:</b> <a href="/cdn-cgi/l/email-protection#78151d14110b0b19380c110b14191a56170a1f">Melissa Haendel</a>.<br><br>
									<i>Funding:</i> This project was funded by NHGRI grant # 5 U01 HG009453 02.</p>

									<h3><a href="http://alaska.dev.eagle-i.net/" class="logo">N-Lighten Network: A Federated Platform for Education Resource Sharing</a></h3>
									<p>Researchers at Harvard University, Oregon Health & Science University and The Ohio State University CTSA Program hubs are developing educational resources, tools and technologies and make them available online to trainees, investigators and other members of the translational scientific team. <b>Contact:</b> <a href="/cdn-cgi/l/email-protection#7c0b1415111d0e3c13140f0952191809">Marijane White</a>.

			<!need to add funding>
									<h3><a href="https://ncit.nci.nih.gov/ncitbrowser/">National Cancer Institute Theasaurus (NCIt)</a></h3>
									<p>The NCI thesaurus (NCIt) is a widely used cancer reference taxonomy that covers over 100,000 terms, developed by the National Cancer Institute (NCI) as a standalone ontology since 2003. The NCI partnered with members of the <a href="https://monarchinitiative.org/">Monarch Initiative</a> to enhance the ontology for interoperability with <a href="http://www.obofoundry.org/">OBO ontologies</a>. <b>Contact:</b> <a href="/cdn-cgi/l/email-protection#90e6f1e3f9fcf5e6e3d0fff8e3e5bef5f4e5">Nicole Vasilevsky.</a>

									<br><br>
									<i>Funding:</i> This project was funded by Leidos contract #15X143
									</p>
			<!need to add funding>
									<h3><a href="https://github.com/openrif" class="logo">OpenRIF</a></h3>
									<p>OpenRIF, the Open Research Information Framework, is an open source community devoted to representing expertise ecosystems - all the things we do and all the things we contribute. The community works on developing and promoting interoperable and extensible semantic infrastructure, such as the VIVO Integrated Semantic Framework (VIVO-ISF), an ontology for representing people, works, and the relationships between them; federated databases modeled on PARDI, the Portfolio Analysis and Reporting Data Infrastructure, for research impact and evaluation;and eagle-i, which aims to make  research resources discoverable via a semantic search interface and represents their relationships to scholarly activities. <b>Contact:</b> <a href="/cdn-cgi/l/email-protection#b4d7dcd5d9c4ddd1c1f4dbdcc7c19ad1d0c1">Robin Champieux.</a>

									<h3><a href="http://bionlp-corpora.sourceforge.net/CRAFT/" class="logo">Annotating the CRAFT Corpus</a></h3>
									<p>The Colorado Richly Annotated Full-Text (CRAFT) Corpus is a collection of 97 full-length, open-access journal articles from the biomedical literature that are manually annotated, for use as gold-standard resources for the training and testing of biomedical Natural Language Processing (NLP) systems. Within these articles, each mention of the concepts explicitly represented in eight prominent Open Biomedical Ontologies (OBOs) has been annotated, resulting in gold-standard markup of genes and gene products, chemicals and molecular entities, biomacromolecular sequence features, cells and cellular and extracellular components and locations, organisms, biological processes and molecular functionalities. With these ~100,000 concept annotations among the ~800,000 words in the 67 articles of the 1.0 release, it is one of the largest gold-standard biomedical semantically annotated corpora. In addition to this substantial conceptual markup, the corpus is fully annotated along a number of syntactic and other axes, notably by sentence segmentation, tokenization, part-of-speech tagging, syntactic parsing, text formatting, and document sectioning. Current efforts are underway to add new annotations. <b>Contact:</b> <a href="/cdn-cgi/l/email-protection#433522302a2f263530032c2b30366d262736">Nicole Vasilevsky.</a><br><br>
									<i>Funding:</i> This project was funded by NIH grants 5R01LM008111 and 5R01LM009254, and DARPA-BAA-14-14.
									</p>

									<h3><a href="https://github.com/OHSU-Library/ohsu-entities">Web Taxology project</a></h3>
									<p>The Web Taxology project is a collaboration between the OHSU Library, Digital Strategy, and the Marketing team to create a data model of all the people, places, and things at OHSU, which is the first step towards improving OHSU's local search results in third-party search engines like Google. The initial goal of the project is to improve patient experience when finding their way to and around OHSU's campuses and clinic locations, with future goals to be used in other contexts and projects throughout the institution as a whole. <b>Contact:</b> <a href="/cdn-cgi/l/email-protection#403728292d2132002f2833356e252435">Marijane White.</a><br><br>
									<i>Funding:</i> This project was supported by the OHSU Library.</p>

			<!need to add funding>
									<h3>Open Insight</h3>
									<p>Open Insight is an education and outreach project designed to stimulate early career researchers' engagement with open science practices through hands-on learning and conversations with leaders in the field. The Open Insight team brings together doctoral students, scientists, and OHSU Library staff with expertise in scholarly communications and data management to explore and promote the practice of open science activities and workflows. <b>Contact:</b> <a href="/cdn-cgi/l/email-protection#cdaea5aca0bda4a8b88da2a5beb8e3a8a9b8">Robin Champieux</a>.

			<!need to add funding>
									<h3>CTAR</h3>
									<p>The Clinical and Translational Activity Reporting (CTAR) tool was a collaboration Oregon Clinical and Translational Research Institute and the OHSU Library's Ontology Development Group to prototype tool that would collocate and analyze data about research activities across a disparate set of internal and external databases (e.g. IRB, grants and contracts, PubMed). Leveraging MeSH, other terminologies, and simple Natural Language Processing (NLP) techniques, the CTAR prototype identified research activity topics and trends, and their classification as clinical or translational. The tool was intended to increase the OHSU's and the Oregon Clinical and Translational Research Institute's ability to strategically contribute to research outcomes and human health. <b>Contact:  </b><a href="/cdn-cgi/l/email-protection#e48981888d979785a4908d97888586ca8b9683">Melissa Haendel</a>.</p>

									<h3><a href="http://ctsaconnect.org/" class="logo">CTSAconnect</a></h3>
									<p>The CTSAconnect project aimed to integrate information about research activities, clinical activities, and scientific resources by creating an Integrated Semantic Framework (ontology). This new framework facilitated the production and consumption of Linked Open Data (a Semantic Web method of sharing data) about investigators, physicians, biomedical research resources, services, and clinical activities. The goal was to enable software to consume data from multiple sources and allow the broadest possible representation of researchers' and clinicians' activities and research products. Current research tracking and networking systems rely largely on publications, but clinical encounters, reagents, techniques, specimens, model organisms, etc., are equally valuable for representing expertise. CTSAconnect was a collaboration between members at OHSU, Stony Brook University, Cornell University, Harvard University, University at Buffalo, and the University of Florida, and leveraged the work of eagle-i, VIVO, and ShareCenter.  <b>Contact:</b> <a href="/cdn-cgi/l/email-protection#0a7c6b7963666f7c794a6562797f246f6e7f">Nicole Vasilevsky.</a><br><br>
									<i>Funding:</i> CTSAconnect was funded by Booz Allen Hamilton grant #CTSA 10-001: 100928SB23.</p>


									<h3><a href="https://www.eagle-i.net/" class="logo">eagle-i Network</a></h3>
									<p>eagle-i is a free application that makes it easy to discover biomedical research resources at a growing network of universities; more than 50,000 resources are listed and more are added every week. Resource types include model organisms, reagents, core laboratory services, instrumentation, and biospecimens. <b>Contact:</b> <a href="/cdn-cgi/l/email-protection#bbd6d8d6cec9c9c2d1fbd4c9dedcd4d5c8cfdacfde95dedfce">Julie McMurry.</a><br><br>
									<i>Funding:</i> eagle-i was funded by Booz Allen Hamilton (Grant # 90177520).</p>

									<h3><a href="https://www.force11.org/group/resource-identification-initiative" class="logo">Resource Identification Initiative</a></h3>
									<p>The Resource Identification Initiative (#RII) was designed to help researchers sufficiently cite the key resources used to produce the scientific findings reported in the biomedical literature. The project aimed to enable resource identification within the biomedical literature through a pilot study promoting the use of unique Research Resource Identifiers (RRIDs). In addition to being unique, RRID’s meet three key criteria, they are: 1) Machine readable; 2) Free to generate and access; 3) Consistent across publishers and journals. A diverse group of collaborators led the project, including the Neuroscience Information Framework and the OHSU Library. <b>Contact:</b> <a href="/cdn-cgi/l/email-protection#4e382f3d27222b383d0e21263d3b602b2a3b">Nicole Vasilevsky.</a></p>
									<i>Funding:</i> The Resource Identification Initiative was supported by the <a href="https://www.nih.gov/">NIH</a> and the <a href="https://neuinfo.org//">INCF.</a></p>

									<h3>Biospecimen Query</h3>
									<p>This project explored options for enhancing search capabilities for an existing biospecimen search application. Text processing tools were used to map anatomy, pathology, and disease concepts from existing terminologies and ontologies to pathology reports that are currently represented in an unstructured natural text form. The concepts identified in the text were also organized in a relational structure to enable taxonomic and parthood based searches. This was a small exploratory project with a goal of integrating these capabilities in an ongoing effort to expand and integrate OHSU's biospecimen databases. <b>Contact:</b> <a href="/cdn-cgi/l/email-protection#2e434b42475d5d4f6e5a475d424f4c00415c49">Melissa Haendel.</a><br><br>
									<i>Funding:</i> This work was funded by OHSU's Medical Research Foundation.</p>



      <!-- Footer -->
      <footer id="footer">
        <div class="inner">
          <div class="flex">
            <div class="copyright">
              &copy; Untitled. Design: <a href="https://templated.co">TEMPLATED</a>. Images: <a href="https://unsplash.com">Unsplash</a>.
            </div>
            <ul class="icons">
              <li><a href="mailto:info@tislabs.org" class="icon fa-envelope"><span class="label">feedback@tislabs.org</span></a></li>
              <li><a href="https://twitter.com/monarchinit" class="icon fa-twitter"><span class="label">Twitter</span></a></li>
            </ul>
          </div>
        </div>
      </footer>

      <!-- Scripts -->
      <script src="assets/js/jquery.min.js"></script>
      <script src="assets/js/skel.min.js"></script>
      <script src="assets/js/util.js"></script>
      <script src="assets/js/main.js"></script>

</body>

</html>
